server/routes/api.js

```diff
@@ -27,6 +27,39 @@ router.get('/asset-compute-endpoint', async function(req, res) {
     });
 });
 
+
+router.get('/get-journal-url', async function(req, res) {
```

Comment:
More an RPC-type endpoint, but OK since existing endpoints in the file have a similar structure (just a FYI).

Again FYI:
I would have called the endpoint `journal`. You get the journal by using the HTTP verb get, and it returns you the journal's url in the response. That would make it a bit more REST-like.